Library/Formula/psql2csv.rb

```diff
@@ -0,0 +1,16 @@
+class Psql2csv < Formula
+  desc "Run a query in psql and output the result as CSV"
+  homepage "https://github.com/fphilipe/psql2csv"
+  url "https://github.com/fphilipe/psql2csv/archive/v0.4.tar.gz"
+  sha256 "da009e8fbba669c32489beea0c5caf4c0a4184030403e8027a5820d8d8ebc536"
+
+  depends_on "postgresql"
+
+  def install
+    bin.install "psql2csv"
+  end
+
+  test do
+    system bin/"psql2csv", "--help"
```

Comment:
Could you write a test for the actual functionality? You could use e.g. `--dry-run` to print the query.